modules/post/multi/manage/zip.rb

```diff
@@ -70,7 +70,7 @@ def steal_token
     rescue Rex::Post::Meterpreter::RequestError => e
       # It could raise an exception even when the token is successfully stolen,
       # so we will just log the exception and move on.
-      elog("#{e.class} #{e.message}\n#{e.backtrace * "\n"}")
+      elog("Exception encountered when attempting to steal Token", error: e)
```

Comment:
This log message contradicts the above comment 🤔 

```suggestion
      elog(e)
```